tests/network/sriov.go

```diff
@@ -250,17 +252,14 @@ var _ = Describe("SRIOV", Serial, decorators.SRIOV, func() {
 			Expect(checkDefaultInterfaceInPod(vmi)).To(Succeed())
 
 			By("checking virtual machine instance has two interfaces")
-			checkInterfacesInGuest(vmi, []string{"eth0", "eth1"})
+			expectedInterfaces := []string{"eth0", "eth1"}
```

Comment:
Small nice to have suggestion:
```suggestion
                        const (
                          virtioIfaceName = "eth0"
                          sriovIfaceName = "eth1"
                        )
			expectedInterfaces := []string{virtioIfaceName, sriovIfaceName}
```

Reply:
Just to understand: We expect both interfaces to be on the root bus?

The classification of what is eth0 and what is eth1 is incorrect. We know the naming is not deterministic.

Reply:
IIUC, according to :https://github.com/kubevirt/kubevirt/blob/df896a54177bafedbff8ab7674b0f316cd79de2a/pkg/virt-launcher/virtwrap/converter/pci-placement.go#L11

We ask libvirt to have both interfaces on the root complex.

Reply:
It was in response to @EdDev's question:
> Just to understand: We expect both interfaces to be on the root bus?

Reply:
At this point, I think we need someone from virt to ack this statement.
This is getting into deep virt territory.

If both interfaces should be set on the root PCI bus, then why it is not the case here?
I am realizing that I do not have the expertise to ack if what is checked is correct or not.
Please check this with someone from sig-compute or libvirt.

/approve cancel